dogs-initial/src/main/java/com/lambdaschool/projectrestdogs/services/MessageSender.java

```diff
@@ -0,0 +1,39 @@
+package com.lambdaschool.projectrestdogs.services;
+import com.lambdaschool.projectrestdogs.Dog;
+//import com.lambdaschool.projectrestdogs.ProjectrestdogsApplication;
```

Comment:
remember what john said? you only need to comment one or two lines of code to disable the messages, you only need to comment out where your calling the send message